target/ppc/translate/vsx-impl.c.inc

```diff
@@ -2156,28 +2170,28 @@ static bool do_ger_XX3(DisasContext *ctx, arg_XX3 *a, int op_flag,
     m.pmsk = 0xFF;
     m.ymsk = 0xF;
     m.xmsk = 0xF;
-    return do_ger_MMIRR_XX3(ctx, &m, op_flag, helper);
-}
-
-TRANS(XVI4GER8    , do_ger_XX3, GER_NOP, gen_helper_xvi4ger8)
-TRANS(XVI4GER8PP  , do_ger_XX3, GER_PP , gen_helper_xvi4ger8)
-TRANS(XVI8GER4    , do_ger_XX3, GER_NOP, gen_helper_xvi8ger4)
-TRANS(XVI8GER4PP  , do_ger_XX3, GER_PP , gen_helper_xvi8ger4)
-TRANS(XVI8GER4SPP , do_ger_XX3, GER_SPP, gen_helper_xvi8ger4)
-TRANS(XVI16GER2   , do_ger_XX3, GER_NOP, gen_helper_xvi16ger2)
-TRANS(XVI16GER2PP , do_ger_XX3, GER_PP , gen_helper_xvi16ger2)
-TRANS(XVI16GER2S  , do_ger_XX3, GER_SAT, gen_helper_xvi16ger2)
-TRANS(XVI16GER2SPP, do_ger_XX3, GER_SPP, gen_helper_xvi16ger2)
-
-TRANS64(PMXVI4GER8    , do_ger_MMIRR_XX3, GER_NOP, gen_helper_xvi4ger8)
-TRANS64(PMXVI4GER8PP  , do_ger_MMIRR_XX3, GER_PP , gen_helper_xvi4ger8)
-TRANS64(PMXVI8GER4    , do_ger_MMIRR_XX3, GER_NOP, gen_helper_xvi8ger4)
-TRANS64(PMXVI8GER4PP  , do_ger_MMIRR_XX3, GER_PP , gen_helper_xvi8ger4)
-TRANS64(PMXVI8GER4SPP , do_ger_MMIRR_XX3, GER_SPP, gen_helper_xvi8ger4)
-TRANS64(PMXVI16GER2   , do_ger_MMIRR_XX3, GER_NOP, gen_helper_xvi16ger2)
-TRANS64(PMXVI16GER2PP , do_ger_MMIRR_XX3, GER_PP , gen_helper_xvi16ger2)
-TRANS64(PMXVI16GER2S  , do_ger_MMIRR_XX3, GER_SAT, gen_helper_xvi16ger2)
-TRANS64(PMXVI16GER2SPP, do_ger_MMIRR_XX3, GER_SPP, gen_helper_xvi16ger2)
+    return do_ger_MMIRR_XX3(ctx, &m, acc, sat, helper);
+}
+
+TRANS(XVI4GER8    , do_ger_XX3, 0, 0, gen_helper_xvi4ger8)
```

Comment:
Here too we can undo the changes and use the enum options (and also in do_ger_XX3).